### PR TITLE
fix: karma ripples dot

### PR DESCRIPTION
**Pull request by yusufputra**
### Summary by CodeRabbit

- Refactor: Updated the `CircleGradient` component in Karma. The SVG circle now renders conditionally based on the `fill` prop. This change enhances the visual representation of data, improving user experience by providing a more accurate and dynamic display when the fill percentage is not at 100%.

### CODE REVIEW

**Comment by bastian-purrer** [comment]
Image description   CodeRabbit



Currently reviewing new changes in this PR...

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 7473e50b0487f87af5f340e6cabe9037892e4a98 and 9f3713570663d5c778996b8a85a656d48d9ddd0f commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/components/Karma/CircleGradient.tsx (1)
</details>






---

File: src/components/Karma/CircleGradient.tsx [hunk 1]
```diff
@@ -45,21 +45,23 @@ export const CircleGradient = (props: {
...
+            cy={props.size / 2}
+            r={r + 1}
+            stroke="#E8EBED"
+            strokeWidth={props.width + 1}
+            strokeDasharray={circleCircumference}
+            strokeDashoffset={`${circleStrokeDashoffset * -1}`}
+            fill="none"
+          />
+        </Svg>
+      )}
```
Comment (asaadam) [inline src/components/Karma/CircleGradient.tsx:64]: apa yg terjadi kalau fill nya 100?

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 7473e50b0487f87af5f340e6cabe9037892e4a98 and 9f3713570663d5c778996b8a85a656d48d9ddd0f commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/components/Karma/CircleGradient.tsx (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/components/Karma/CircleGradient.tsx

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 7473e50b0487f87af5f340e6cabe9037892e4a98 and 9f3713570663d5c778996b8a85a656d48d9ddd0f commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/components/Karma/CircleGradient.tsx (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/components/Karma/CircleGradient.tsx

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 7473e50b0487f87af5f340e6cabe9037892e4a98 and 9f3713570663d5c778996b8a85a656d48d9ddd0f commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/components/Karma/CircleGradient.tsx (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/components/Karma/CircleGradient.tsx

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

File: src/components/Karma/CircleGradient.tsx (hunk 1 above)
Comment (yusufputra) [reply src/components/Karma/CircleGradient.tsx:64]: abu abu nya hilang bwang, jadi full circle berwarna, somehow kalo karma pointnya 100, dia masih munculin abu abu tipis banget, jadi aku remove lingkaran abu abu nya kalo 100, biar muncul yang warna aja

Comment (asaadam) [reply src/components/Karma/CircleGradient.tsx:64]: oalah buat sisanya ya. 
sedangkan kalau nilainya udh 100, abu" ga usah di render,

ok acc

**Review by asaadam — APPROVED** [review]
bagus

**Pull request merged**